orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/functions/HelloExpressionFunctionProvider.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.orca.pipeline.expressions.functions;
+
+import com.netflix.spinnaker.kork.expressions.ExpressionFunctionProvider;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.pf4j.Extension;
+
+@Extension
```

Comment:
So this would essentially be a "system extension"?

Reply:
That's right. Although I'm gonna delete this from the PR. I didn't mean for it to come along for the ride.

Reply:
Yeah, I have no intention of changing how spel function customization is done, only in providing new ways to provde said functions.